src/graphql/fragments/postsQueryFragment.js

```diff
@@ -4,14 +4,18 @@ const postsQueryFragment = `
 posts(
   activePostsOnly: $activePostsOnly,
   afterTime: $afterTime,
+  announcementsOnly: $announcementsOnly,
```

Comment:
one thing to test is because of the way we share this fragment, do we need to add these parameters to more places. test the Recent Activity on the user profile and also the map

Reply:
Do we need to add them to other place? If they aren't passed in, they won't be factored into the query, no?

Reply:
well just test the use profile (RecentActivity component) and map and see if they work